api/api/specs/routers.yaml

```diff
@@ -512,8 +512,6 @@ components:
           type: "array"
           items:
             $ref: "#/components/schemas/Route"
-        default_route:
```

Comment:
I don't think this was ever meant to be there, it should have only been `default_route_id`. Removing it.